package-system/AWSGameLiftServerSDK/build_package_image.py

```diff
@@ -155,13 +160,14 @@ def get_custom_build_env():
     return None
 
 # configure gamelift server sdk project
-def configure_sdk_project(source_folder: str,
+def configure_sdk_project(working_directory: WorkingDirectoryInfo,
                           build_folder: str,
                           build_type: str,
                           lib_type: str) -> None:
+    source_folder: str = working_directory.source_path.resolve()
     build_shared: str = "ON" if lib_type == "Shared" else "OFF"
     if PACKAGE_PLATFORM == PACKAGE_PLATFORM_OPTIONS[0]:
-        generator: str = "-G \"Visual Studio 15 2017\" -A x64"
+        generator: str = "-G \"Visual Studio 17 2022\""
```

Comment:
The year isn't not needed in the generator.
"Visual Studio 17" works just as well in cmake
```suggestion
        generator: str = "-G \"Visual Studio 17\""
```

Reply:
Sounds good I'll update that, thanks!